### PR TITLE
Rename protocol modules

### LINKED ISSUE #1922: Rename protocol modules in emoji and test-app
Should be 'protocol-guest' and 'protocol-host' instead of 'compose/protocol' and 'widget/protocol'.

### PR DESCRIPTION
**Pull request by ahmed3elshaer**
This PR includes renaming operations to standardize the naming convention across the scheme protocols:

- Renamed `sample emoji scheme widget/protocol` and `compose/protocol` to `protocol-host` and `protocol-guest`.
- Updated the test app naming from `widget-protocol` to `protocol-host`.
- Changed the test app `compose-protocol` to `protocol-guest`.

Fixes [issue #1922](https://github.com/cashapp/redwood/issues/1922)

---
- [ ] `CHANGELOG.md`'s "Unreleased" section has been updated, if applicable.

### CODE REVIEW

**Comment by ahmed3elshaer** [comment]
does this require updates in the changelog ?

**Comment by JakeWharton** [comment]
> does this require updates in the changelog ?

No, since it doesn't effect end-users.

File: settings.gradle [hunk 1]
```diff
@@ -86,11 +86,11 @@ include ':redwood-yoga'
 
 include ':test-app:schema'
 include ':test-app:schema:compose'
-include ':test-app:schema:compose-protocol'
+include ':test-app:schema:protocol-guest'
 include ':test-app:schema:modifiers'
 include ':test-app:schema:testing'
 include ':test-app:schema:widget'
-include ':test-app:schema:widget-protocol'
+include ':test-app:schema:protocol-host'
```
Comment (JakeWharton) [inline settings.gradle]: Can you keep this and below sorted alphabetically?

**Review by JakeWharton — APPROVED** [review]

**Comment by JakeWharton** [comment]
Thanks for doing this!

**Comment by ahmed3elshaer** [comment]
> Thanks for doing this!

I'm glad that I can be of help!

File: settings.gradle (hunk 1 above)
Comment (ahmed3elshaer) [reply settings.gradle]: ✅

**Pull request merged**